### PR TITLE
Docs: removing callouts and other updates

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?
This PR makes the following changes:

- Removes callouts _"This documentation is currently under development."_ from the getting started track: [preview](https://faststore-site-git-docs-callout-faststore.vercel.app/docs/getting-started/0-getting-started-overview)
- Fixes the titles of the steps in _Creating a new section_ doc: [preview](https://faststore-site-git-docs-callout-faststore.vercel.app/docs/component-customization/creating-a-new-section)
- Adds content to the _Next steps_ doc: [preview](https://faststore-site-git-docs-callout-faststore.vercel.app/docs/getting-started/5-next-steps)
- Adds content to _Setting up a VTEX account for FastStore_ doc: [preview](https://faststore-site-git-docs-callout-faststore.vercel.app/docs/go-live/1-setting-up-vtex-account-for-faststore)

### CODE REVIEW

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 1]
```diff
@@ -14,3 +14,25 @@ import { Callout } from 'nextra-theme-docs'
...
+
+Explore the various ways you can extend and customize the components of your store by checking the [Component Customization track](/docs/component-customization/component-customization-overview). This could involve modifying existing components or creating new ones to meet your specific requirements. By customizing components, you can tailor your store's functionality to suit your business needs.
+
+## UI components
+
+Enhance your store's features with the [FastStore UI Components](/components). These components are designed to help you quickly implement and customize features in your store.
+
+## Go-live
+
+Integrate your FastStore project with VTEX and get ready to launch your store with the [Go-live track](/docs/go-live/overview) . This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless checkout experience for your customers.
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:38]:
```suggestion
Integrate your FastStore project with VTEX and get ready to launch your store with the [Go-live track](/docs/go-live/overview). This involves connecting your store to VTEX's infrastructure, setting up payment gateways, configuring shipping options, and ensuring a seamless checkout experience for your customers.
```

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 2]
```diff
@@ -14,3 +14,25 @@ import { Callout } from 'nextra-theme-docs'
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
 </Callout>
+
+Now that you have your FastStore project up and running, it's important to explore the additional resources to enhance your store experience:
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:18]: Cool! 👍

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 3]
```diff
@@ -14,3 +14,25 @@ import { Callout } from 'nextra-theme-docs'
 <Callout type="warning" emoji="⚠️">
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:14]: I think we can remove this callout.

**Review by hellofanny — APPROVED** [review]

File: apps/site/pages/docs/getting-started/5-next-steps.mdx (hunk 3 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/getting-started/5-next-steps.mdx:14]: Good point, Fanny ;)

Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:14]:
```suggestion
```

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 4]
```diff
@@ -14,3 +14,25 @@ import { Callout } from 'nextra-theme-docs'
...
 This documentation is currently under development.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:15]:
```suggestion
```

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 5]
```diff
@@ -14,3 +14,25 @@ import { Callout } from 'nextra-theme-docs'
...
 </Callout>
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:16]:
```suggestion
```

File: apps/site/pages/docs/getting-started/5-next-steps.mdx [hunk 6]
```diff
@@ -11,6 +11,25 @@ import { Callout } from 'nextra-theme-docs'
 
 </header>
 
-<Callout type="warning" emoji="⚠️">
-This documentation is currently under development.
-</Callout>
+
+Now that you have your FastStore project up and running, it's important to explore the additional resources to enhance your store experience:
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/5-next-steps.mdx:15]:
```suggestion
Now that you have your FastStore project up and running, it is important to explore the additional resources to enhance your store experience:
```

**Pull request merged**